### ISSUE TITLE
TTY output should include another colon

### ISSUE THREAD

**Issue by biffen** [hadolint/hadolint#766]
- [ ] This is a bug report
- [X] This is a feature request
- [X] I searched existing issues before opening this one

### Expected behavior

This is such a minor thing but it would make such a nice change.

When using Hadolint from Emacs (with `tty` output) I expect Hadolint to follow the file-colon-line-colon convention (I don’t know that there is a name for this convention, but GNU describes it [here](https://www.gnu.org/prep/standards/html_node/Errors.html)) so that Emacs, and probably other tools (IIRC the Atom editor has something similar), can identify the errors. That way each error becomes a link of sorts to the file and line where the error occurred.

(And yes, I know I can configure Emacs to match more patterns, but there’s really nothing specific to match, so I’d rather see this changed in Hadolint.)

### Actual behavior

The last colon is missing and Emacs doesn’t see the errors as errors.

### Steps to reproduce the behavior

```sh
$ hadolint - <<EOF
FROM alpine
EOF
-:1 DL3006 warning: Always tag the version of an image explicitly
```

If the above contained `-:1:` I’d be happier.

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version` or
  `docker run --rm ghcr.io/hadolint/hadolint hadolint --version`:**

```bash
Haskell Dockerfile Linter UNKNOWN
```

**Dockerfile** (if relevant)

Not relevant (I think, let me know otherwise).

**Additional environment details (OS, stack version, etc.)**

Not relevant (again, let me know if I’m wrong)

---

BTW, thanks for an excellent tool!

**Comment by biffen** [comment]
Awesome! Thanks!
